include/wx/textctrl.h

```diff
@@ -608,6 +608,69 @@ class WXDLLIMPEXP_CORE wxTextAttr
     wxString            m_listStyleName;
 };
 
+// ----------------------------------------------------------------------------
+// Search features for wxTextCtrl
+// ----------------------------------------------------------------------------
+
+enum class wxTextSearchDirection
+{
+    Down,
+    Up
+};
+
+// search options
+// --------------
+struct wxTextSearch
+{
+    wxTextSearch(const wxString& text) : m_searchValue(text) {}
```

Comment:
Minor, but the ctor should be explicit we probably also want to provide the default ctor, especially as we have `SearchValue()`, so:

```suggestion
    explicit wxTextSearch(const wxString& text = wxString{}) : m_searchValue(text) {}
```